CHANGELOG.md

```diff
@@ -4,6 +4,48 @@ All notable changes to this project will be documented in this file.
+
+You can use ODP, a high-performance [Customer Data Platform (CDP)](https://www.optimizely.com/optimization-glossary/customer-data-platform/), to easily create complex real-time segments (RTS) using first-party and 50+ third-party data sources out of the box. You can create custom schemas that support the user attributes important for your business, and stitch together user behavior done on different devices to better understand and target your customers for personalized user experiences. ODP can be used as a single source of truth for these segments in any Optimizely or 3rd party tool.
+
+With ODP accounts integrated into Optimizely projects, you can build audiences using segments pre-defined in ODP. The SDK will fetch the segments for given users and make decisions using the segments. For access to ODP audience targeting in your Feature Experimentation account, please contact your Optimizely Customer Success Manager.
+
+This version includes the following changes:
+
+- `FetchQualifiedSegments()` API has been added to the `/decide` endpoint. This API will retrieve user segments from the ODP server. The fetched segments will be used for audience evaluation. Fetched data will be stored in the local cache to avoid repeated network delays.
+
+- `SendOdpEvent()` API has been added with the `/send-opd-event` endpoint. Customers can build/send arbitrary ODP events that will bind user identifiers and data to user profiles in ODP.
+
+For details, refer to our documentation pages:
+
+* [Advanced Audience Targeting](https://docs.developers.optimizely.com/feature-experimentation/docs/optimizely-data-platform-advanced-audience-targeting) 
+
+* [Server SDK Support](https://docs.developers.optimizely.com/feature-experimentation/docs/advanced-audience-targeting-for-server-side-sdks)
+
+* [Use Optimizely Agent](https://docs.developers.optimizely.com/feature-experimentation/docs/use-optimizely-agent)
+
+* [Configure Optimizely Agent](https://docs.developers.optimizely.com/feature-experimentation/docs/configure-optimizely-agent)
+
+### Breaking Changes
+
+- ODPManager in the SDK is enabled by default. Unless an ODP account is integrated into the Optimizely projects, most ODPManager functions will be ignored. If needed, ODPManager can be disabled when OptimizelyClient is instantiated. From Agent, it can be switched off from config.yaml or env variables.
+- Updated go-sdk version to v2.0.0 with module path github.com/optimizely/go-sdk/v2
+
+### Functionality Enhancement
+* Updated openapi schema to 3.1.0. ([#392](https://github.com/optimizely/agent/pull/392))
+* Added support for prometheus metrics. ([#348](https://github.com/optimizely/agent/pull/348))
+* Github Issue template is udpated. ([#396](https://github.com/optimizely/agent/pull/396))
+* Updated go version to 1.21. ([#398](https://github.com/optimizely/agent/pull/398))
+* Added OpenTelemetry Tracing Support. ([#400](https://github.com/optimizely/agent/pull/400))
```

Comment:
This should be listed as "New Fatures"? Notification consistency and datafile syncer also?

Reply:
Actually Notification consistency was a bug, updated accordingly, Please check